screens/auth/SignUpScreen.js

```diff
@@ -40,7 +61,13 @@ export default class SignUp extends React.Component {
       passwordError: '',
       phoneNumber: '',
       phoneNumberError: '',
-      token: ''
+      token: '',
+      indicators: {
+        [signUpFields.NAME]: [fieldStateColors.INACTIVE],
+        [signUpFields.PHONENUM]: [fieldStateColors.INACTIVE],
+        [signUpFields.PASSWORD]: [fieldStateColors.INACTIVE]
+      },
+      signUpPermission: false
```

Comment:
Could you comment out line 77 in `componentDidMount` (`this.registerForPushNotificationsAsync()` (but don't remove because we have to fix it ;_;)